src/lib/profiles/security/WeaveCASEEngine.cpp

```diff
@@ -681,66 +684,75 @@ WEAVE_ERROR WeaveCASEEngine::AppendSignature(BeginSessionMessageBase& msg, Packe
 
 // Verify a CASE message signature (for the message in the supplied buffer) against a given public key.
 // Returns a hash of the signed portion of the message in the supplied bufer.
-WEAVE_ERROR WeaveCASEEngine::VerifySignature(BeginSessionMessageBase& msg, PacketBuffer *msgBuf, uint8_t *msgHash)
+WEAVE_ERROR WeaveCASEEngine::VerifySignature(BeginSessionContext & msg, PacketBuffer * msgBuf, uint8_t * msgHash)
```

Comment:
msg --> msgCtx

Reply:
Agreed.  I am holding off making this change until people are done reviewing because it creates a lot of noise in the diffs, obscuring the substantive changes.